### PR TITLE
Add HealthKit, MediaPlayer, MetricKit, PhotoKit frameworks

### PR DESCRIPTION
**Pull request by silvanshade**
Supersedes #404, #406, #407, #405.

### CODE REVIEW

**Review by madsmtm — CHANGES_REQUESTED** [review]
Could I get you to fix the [errors that `header-translator` outputs](https://github.com/madsmtm/objc2/actions/runs/4239297384/jobs/7367272758#step:5:315)? (CI should really be able to catch this, but I haven't yet found a good way to go from `error!` to an `ExitCode::FAILURE` in `main`).

Also, a changelog entry would be nice.

**Comment by silvanshade** [comment]
Errors should be fixed now and I added a CHANGELOG entry.

File: crates/icrate/CHANGELOG.md [hunk 1]
```diff
@@ -56,60 +63,65 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
   - `UniformTypeIdentifiers`
   - `UserNotifications`
   - `WebKit`
-* Updated the SDK version from XCode `14.0.1` to `14.2`.
+- Updated the SDK version from XCode `14.0.1` to `14.2`.
```
Comment (madsmtm) [inline crates/icrate/CHANGELOG.md:66]: I appreciate the desire to make the changelogs more "standard", but please do so in a separate PR (and do it for the other changelogs as well).

**Review by madsmtm — CHANGES_REQUESTED** [review]

File: crates/icrate/CHANGELOG.md (hunk 1 above)
Comment (silvanshade) [reply crates/icrate/CHANGELOG.md:66]: Oh, that was an unintentional format-on-save that I didn't notice. Sorry.

File: crates/icrate/CHANGELOG.md [hunk 2]
```diff
@@ -10,6 +10,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## icrate Unreleased - YYYY-MM-DD
 
+* Added the following frameworks:
+  - `HealthKit`
+  - `MediaPlayer`
+  - `MetricKit`
+  - `PhotoKit`
```
Comment (madsmtm) [inline crates/icrate/CHANGELOG.md:18]: Nit: Missing header

```suggestion
### Added
* Added the following frameworks:
  - `HealthKit`
  - `MediaPlayer`
  - `MetricKit`
  - `PhotoKit`


```

**Review by madsmtm — APPROVED** [review]
Thanks! (I'll merge once CI finishes)
